content/news/2024-03-11-bevy-foundation/index.md

```diff
@@ -0,0 +1,161 @@
++++
+title = "Bevy Foundation"
+date = 2024-03-11
+[extra]
+image = "bevy_foundation.svg"
+# show_image = true
+author = "Carter Anderson"
+twitter = "cart_cart"
+github = "cart"
+youtube = "cartdev"
++++
+
+It is with excitement that I unveil to the world ...
+
+<img src="bevy_foundation.svg" alt="Bevy Foundation Logo" style="margin-top: 2rem; margin-bottom: 2rem;" />
+
+The [**Bevy Foundation**](/foundation) is the next step in our journey to build a world class free and open source game engine. **Bevy Foundation** is a non-profit organization formed in Washington State, with a pending federal 501(c)(3) tax-exemption application (more on this later).
+
+<!-- more -->
```

Comment:
Is this here because you want to add more content or is it just a dead comment?

Reply:
This is a Special Zola Thing that determines the cutoff for the description in `/news` card for this post